http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -493,6 +504,18 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       else appQuery.updateStatus(dbApp.app.id, AppStatus.Error).transaction
     } yield ()
 
+  def createGalaxyPostresDisk(project: GoogleProject, zone: ZoneName, appName: AppName): Disk =
+    Disk
+      .newBuilder()
+      .setName(s"${appName.value}-${config.galaxyAppConfig.postgresDiskNameSuffix}")
+      .setZone(zone.value)
+      .setSizeGb(config.galaxyAppConfig.postgresDiskSize.gb.toString)
+      .setPhysicalBlockSizeBytes("4096")
```

Comment:
Should the value `4096` come from a config?

Reply:
I think it's on a disk config, I'll look that up